libsplinter/src/auth/oauth/builder/mod.rs

```diff
@@ -144,4 +154,14 @@ impl OAuthClientBuilder {
         self.identity_provider = Some(identity_provider);
         self
     }
+
+    /// Sets the in-flight request store to use to store values between requests to and from the
```

Comment:
I think "to use" should be removed from this comment?

Reply:
same for other instance of with_inflight_request_store

Reply:
Reworded in all places.